lib/features/home_screen/home_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/goals/cubit/goal_tracker_cubit.dart';
+import 'package:givt_app_kids/features/goals/pages/goal_screen.dart';
+import 'package:givt_app_kids/features/history/history_logic/history_cubit.dart';
+import 'package:givt_app_kids/features/history/history_screen.dart';
+import 'package:givt_app_kids/features/home_screen/cubit/navigation_cubit.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/features/profiles/screens/profile_screen.dart';
+import 'package:givt_app_kids/features/home_screen/widgets/custom_navigation_bar.dart';
+import 'package:givt_app_kids/features/home_screen/widgets/home_screen_app_bar.dart';
+import 'package:givt_app_kids/helpers/analytics_helper.dart';
+
+class HomeScreen extends StatefulWidget {
+  const HomeScreen({super.key});
+
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  int currentPageIndex = 0;
```

Comment:
why do we need this member (and stateful widget in general) if we already have NavigationCubit?

Reply:
Leftover from the material design navigation bar implementation, removed ✅